logparser/logs/debug/logs.py

```diff
@@ -278,8 +278,11 @@ def get_regex_list():
     regex.append([debug.on_ignored_message,
                   r"DDS_DomainParticipantFactory_initializeI:Welcome to NDDS"])
     regex.append([debug.on_ignored_message,
-                  r"DDS_DiscoveryQosPolicy_get_default:no environment" +
-                  r"variable or file NDDS_DISCOVERY_PEERS"])
+                  r"DDS_DiscoveryQosPolicy_get_default:"])
```

Comment:
I think it's not a good idea to ignore all the warnings from a function since this function could print warnings of interest.